Practice3/Decision1.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Practice3
+{
+    public class Decision1
+    {
+        List<int> _list = new List<int>();
+
+        public int MinRange { get; set; }
+        public int MaxRange { get; set; }
+
+        public void SetRange()
+        {
+            Console.WriteLine("Plase set min and max value that can be added into the array.");
+            Console.Write("Min Value: ");
+
+            bool success = false;
+            MinRange = Int32.MinValue;
```

Comment:
please, change the Int32 to int everywhere
![image](https://user-images.githubusercontent.com/9382934/118265552-0413e580-b4c2-11eb-982a-664d0842094e.png)